src/main/java/cms/gov/madie/measure/models/MeasureMetaData.java

```diff
@@ -7,4 +7,5 @@
 @Data
 public class MeasureMetaData {
   private String measureSteward;
+  private String measureDescription;
```

Comment:
@gregory-akins / @jkotanchik-SB can we add a task to refactor some of these fields? e.g `measureSteward` this can be just `steward`. It is quite clear that this is measure steward because this is in MeasureMetaData.

Reply:
@adongare @gregory-akins @jkotanchik-SB Thanks a lot Ashok! I think it makes a lot of sense to me. I was thinking the same but used measureDescription just to match measureSteward, but if everybody agrees, I can refactor them now. Just let me know, thanks!

Reply:
thanks for updating this. Looks good to me!